rules/models.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import unicode_literals
```

Comment:
please hang a story on the backlog about this item and add a design task to said story. this needs to apply to everything, not just certs.
the structure should be rule_name, rule_applies_to(foreign key), rule_type(enum: exists, interval, missing), etc, etyc